src/lava/magma/runtime/message_infrastructure/message_infrastructure/csrc/channel/grpc/grpc_port.cc

```diff
@@ -117,6 +118,7 @@ void GrpcRecvPort::Join() {
     done_ = true;
     service_ptr_->Stop();
     server_->Shutdown();
+    GetGrpcManager().Release(url_);
```

Comment:
i think it is useless.
the child process invokes this Join func, not parent process.

Reply:
Yes, I tried it, release action doesn't help